packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -610,23 +610,35 @@ export class GarbageCollector implements IGarbageCollector {
             return baseGCDetailsMap;
         });
 
+        // Log all the GC options and the state determined by the garbage collector. This is interesting only for the
+        // summarizer client since it is the only one that runs GC. It also helps keep the telemetry less noisy.
+        const gcConfigProps = JSON.stringify({
+            gcEnabled: this.gcEnabled,
+            sweepEnabled: this.sweepEnabled,
+            runGC: this.shouldRunGC,
+            runSweep: this.shouldRunSweep,
+            writeAtRoot: this._writeDataAtRoot,
+            testMode: this.testMode,
+            sessionExpiry: this.sessionExpiryTimeoutMs,
+            inactiveTimeout: this.inactiveTimeoutMs,
+            existing,
+            ...this.gcOptions,
+        });
+        if (this.isSummarizerClient) {
+            this.mc.logger.sendTelemetryEvent({
+                eventName: "GarbageCollectorLoaded",
+                gcConfigs: gcConfigProps,
```

Comment:
@markfields Can you check if this is what you had in mind?

Reply:
i don't know if this is the same, but i think we are trying to unify on a details field that will be all the details for the message in a json payload

Reply:
Yeah, mark and I were discussing that too. Maybe we can update this later to whatever field name is decided?